src/inflators/image-loader.ts

```diff
@@ -16,5 +20,8 @@ export function inflateImageLoader(world: HubsWorld, eid: number, params: ImageL
     isObjectMenuTarget: false
   });
 
-  // TODO: Use projection
+  const { projection, alphaMode, alphaCutoff } = params;
+  MediaImage.projection[eid] = APP.getSid(projection !== undefined ? projection : ProjectionMode.FLAT);
+  MediaImage.alphaMode[eid] = APP.getSid(alphaMode !== undefined ? alphaMode : AlphaMode.Opaque);
+  MediaImage.alphaCutoff[eid] = alphaCutoff !== undefined ? alphaCutoff : 0.5;
```

Comment:
An entity is assumed that it has `MediaImage` component? I don't think we should assume an entity in inflators alerady has certain components.

Reply:
https://github.com/mozilla/hubs/pull/6111#discussion_r1288496997